std/rfc3999tounix.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package std
+
+import (
+	"time"
+
+	p "github.com/pulumi/pulumi-go-provider"
+	"github.com/pulumi/pulumi-go-provider/infer"
+)
+
+type Rfc3339tounix struct{}
```

Comment:
```suggestion
type Rfc3339ToUnix struct{}
```
CamelCase all the names

Reply:
Other functions are also using the sentence case like
https://github.com/pulumi/pulumi-std/blob/4e9e486d05bde8b5d7b2a252a5bf50687e7d6fd1/std/base64decode.go#L26
https://github.com/pulumi/pulumi-std/blob/4e9e486d05bde8b5d7b2a252a5bf50687e7d6fd1/std/cidrnetmask.go#L24
https://github.com/pulumi/pulumi-std/blob/4e9e486d05bde8b5d7b2a252a5bf50687e7d6fd1/std/filebase64sha256.go#L22

and others, hence I used that casing here. I also tried naming `RFC3999ToUnix` and that resulted in some tokens being `rfC3999ToUnix` which is weird.